osbuild/util/mnt.py

```diff
@@ -44,14 +44,13 @@ class MountGuard(contextlib.AbstractContextManager):
     def __init__(self):
         self.mounts = []
 
-    def mount(self, source, target, bind=True, ro=False, rw=False, mode="0755"):
+    def mount(self, source, target, bind=True, permissions=None, mode="0755"):
         options = []
         if bind:
             options += ["bind"]
-        if ro:
-            options += ["ro"]
-        if rw:
-            options += ["rw"]
+        if permissions:
+            assert permissions in ["rw", "ro"], f"unknown filesystem permissions: {permissions}"
```

Comment:
I think that it would make more sense to raise a `ValueError`, instead of asserting.